.github/workflows/pull_request_full_build.yml

```diff
@@ -4,7 +4,8 @@ on:
   pull_request:
     branches:
       - master
-
+      - java21
+      - nutcracker
```

Comment:
Do we still need these branches? Cant we remove them?

Reply:
yes. will remove them